docker-compose-web-modeler.yaml

```diff
@@ -76,6 +76,7 @@ services:
     environment:
       JAVA_OPTIONS: -Xmx128m
       LOGGING_LEVEL_IO_CAMUNDA_MODELER: DEBUG
+      CAMUNDA_IDENTITY_BASE_URL: http://identity:8084/
```

Comment:
Given the configuration property `camunda.identity.base-url` I think the [recommended format](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#features.external-config.typesafe-configuration-properties.relaxed-binding.environment-variables) for the environment variable would be `CAMUNDA_IDENTITY_BASEURL` (it's also used in the C8 [Helm chart](https://github.com/camunda/camunda-platform-helm/blob/e5d5dbcdba68692a50e31bcd41ffff6d30d25de7/charts/camunda-platform/templates/camunda/configmap-identity-auth.yaml#L9)). However, the variant with the underscore seems to work fine as well.